xml/obs_admin_appliance.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE chapter PUBLIC
+  "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.docbook.org/xml/4.5/docbookx.dtd"
+[
+  <!ENTITY % entities SYSTEM "entity-decl.ent">
+  %entities;
+]>
+<section id="cha.obs.admin.appliance">
+ <title>How to use the OBS appliances in production</title>
+ <para>
+   Generic overview how to use the OBS appliances in production. The appliances can be just
+run executed for testing purposes, but in production use you may want to setup it in a way 
+where production data is seperated from the appliance. A newer version of the appliance 
+can be used just by replacing the system image and it will migrate your user data.
+
+This can be achieved by createing a LVM volume group with the name "OBS". This volume group should be as big as possible because it get's used by the OBS server components for data storage and the OBS workers for root/swap/cache file systems. Prepare a partition of type "8e" and create the LVM via
+
+<screen><command>pvcreate /dev/sdX1
+vgcreate "OBS" /dev/sdX1}}
+</command></screen>
+
+All images come pre-configured with the right set of repositories and can updated via the system tools YaST or zypper at any time. 
+Another way is to replace the entire image. If you update the image keep in mind that you need to have have your data directory (/srv/obs) on a separate storage (LVM volume, partition etc.) otherwise it will get deleted.
+ </para>
+ <section>
+  <title>Server Appliance</title>
+  <para>
+The OBS server images contain a recent openSUSE distribution with a pre-installed and pre-configured OBS backend, API, web frontend and worker.
+
+The system adapts to the hardware on first boot and defaults to automatic IP and DNS configuration via DHCP. The image scans for a LVM volume group called "OBS" and will use that to set up logical volumes for the workers root/swap/cache file systems. Additionally if this volume group contains a logical volume named "server" it will be used as the data partition for the server.
+<screen><command>lvcreate "OBS" -n "server"
+mkfs.xfs /dev/OBS/server
+</command></screen>
+
+
+  </para>
+  <section>
+   <title>Installation Images</title>
+   <para>
+There are installation images. When you boot, they run a basic installer that lets you choose the block device to deploy the image to.
+
+http://openbuildservice.org/download/
+
+You put this image on a USB stick and boot from it.
+
+<screen><command> xzcat obs-server-install.x86_64.raw.xz > /dev/sdX </command></screen>
+
+Warning: /dev/sdX is the main device of your USB stick. Do NOT put it into a partition like /dev/sda1.
```

Comment:
There are warning elements we could put it into. That would highlight it in red and make it more visible.